hypershift-operator/controllers/hostedcluster/hostedcluster_webhook.go

```diff
@@ -221,20 +252,48 @@ func (v nodePoolValidator) validateCreateKubevirtNodePool(ctx context.Context, n
 		return nil, fmt.Errorf("failed to retrive HostedCluster %s/%s; %w", np.Namespace, np.Spec.ClusterName, err)
 	}
 
-	return kvValidator.validate(ctx, v.client, hc)
+	return kvValidator.validate(ctx, hc, np.Spec.Release.Image)
+}
+
+func (v nodePoolValidator) validateUpdateKubevirtNodePool(ctx context.Context, oldNP, newNP *hyperv1.NodePool) error {
+	err := validateJsonAnnotation(newNP.Annotations)
+	if err != nil {
+		return err
+	}
+
+	if oldNP.Spec.Release.Image != newNP.Spec.Release.Image {
+		hc := &hyperv1.HostedCluster{}
+		err = v.client.Get(ctx, client.ObjectKey{Name: newNP.Spec.ClusterName, Namespace: newNP.Namespace}, hc)
+		if err != nil {
+			return fmt.Errorf("failed to retrive HostedCluster %s/%s; %w", newNP.Namespace, newNP.Spec.ClusterName, err)
+		}
+
+		err = kvValidator.validateReleaseImage(ctx, hc, newNP.Spec.Release.Image)
+		if err != nil {
+			return err
+		}
```

Comment:
on create we give this ctx an explicit timeout, but on update we do not. 
example, below is the ctx used during create. 
```
	if _, isTimeout := ctx.Deadline(); !isTimeout {
		var cancel context.CancelFunc
		ctx, cancel = context.WithTimeout(ctx, time.Second*10)
		defer cancel()
	}
```